### DISCUSSION TITLE
Enlarging the Font Size in the Annotation Editor Window

### DISCUSSION THREAD

**Discussion by Rukkha1024** [sumatrapdfreader/sumatrapdf#4160] [Q&A]
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/128712657/081105b7-b542-471e-963f-a6700981df54)

Hello, I'm a user of Sumatra PDF. I'm reaching out to inquire if there's a way to increase the font size within the annotation editor itself. The font size in the editing window on my laptop is too small, making it difficult to read. I would greatly appreciate it if you could inform me of any possible solutions.

**Comment by kjk** [comment]
I've added `UIFontSize` advanced setting in pre-release (https://www.sumatrapdfreader.org/prerelease), build 15991 or higher.

See https://www.sumatrapdfreader.org/docs/Advanced-options-settings for how to change advanced setting.

If it's <9, we will use system font size. If it's >= 9, that's going to be font size used for UI, including edit annotations window.

On my computer default is 12 so try 14 or 16 or whatever works.

You have to restart the app after changing `UIFontSize` setting.

**Reply by Rukkha1024** [reply]
excellent! It really works. 
by the way, user for 64 bit version installer should add `UIFontSize = ` setting in the advanced setting text file. 
great pleasure thanks
